src/main/java/org/skriptlang/skript/bukkit/misc/events/EvtPlayerPickItem.java

```diff
@@ -0,0 +1,157 @@
+package org.skriptlang.skript.bukkit.misc.events;
+
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.classes.data.DefaultComparators;
+import ch.njol.skript.entity.EntityData;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.registrations.EventConverter;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.skript.util.Patterns;
+import ch.njol.skript.util.slot.InventorySlot;
+import ch.njol.skript.util.slot.Slot;
+import io.papermc.paper.event.player.PlayerPickBlockEvent;
+import io.papermc.paper.event.player.PlayerPickEntityEvent;
+import io.papermc.paper.event.player.PlayerPickItemEvent;
+import org.bukkit.block.Block;
+import org.bukkit.block.data.BlockData;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.registration.BukkitSyntaxInfos;
+import org.skriptlang.skript.docs.Origin;
+import org.skriptlang.skript.lang.comparator.Relation;
+import org.skriptlang.skript.registration.SyntaxRegistry;
+
+public class EvtPlayerPickItem extends SkriptEvent {
+
+	private static final Patterns<PickType> PATTERNS = new Patterns<>(new Object[][]{
+		{"[player] pick[ing] [of] [an|any] item", PickType.ANY},
+		{"[player] pick[ing] [of] [a|any] block", PickType.BLOCK},
+		{"[player] pick[ing] [of] [an|any] entity", PickType.ENTITY},
+		{"[player] pick[ing] [of] %entitydata/itemtype/blockdata%", null}
+	});
+
+	public static void register(SyntaxRegistry registry, Origin origin) {
+		registry.register(BukkitSyntaxInfos.Event.KEY, BukkitSyntaxInfos.Event.builder(EvtPlayerPickItem.class, "Player Pick Item")
+			.supplier(EvtPlayerPickItem::new)
+			.origin(origin)
+			.addEvent(PlayerPickBlockEvent.class)
+			.addEvent(PlayerPickEntityEvent.class)
+			.addPatterns(PATTERNS.getPatterns())
+			.addDescription("Called when a player picks an item, block or an entity" + 
+					" using the pick block key (default middle mouse button).",
+				"The past event-slot represents the slot in which the item that will be put into the players hotbar is located." +
+					" Or nothing, if the item is not in the inventory. (May be changed)",
+				"The event-slot represents the slot in the hotbar where the picked item will be placed. (May be changed)")
```

Comment:
```suggestion
				"The event-slot represents the slot in the hotbar where the picked item will be placed.",
				"Both event-slots may be set to new slots.)
```

just some clarity so users don't think the description may be changed.